README.md

```diff
@@ -73,7 +73,7 @@ When starting to work on an existing project (a project we just cloned from gith
 
 Afther that, we start kicad (after source the setup.bash)
 
-1. `$ dockerr_kicad`
+1. `$ docker_kicad`
```

Comment:
This is actually old. Now the kicad command is called <project_name>_kicad instaed of docker_kicad. Can you please update it (using a nice notation for the 'name of the project'. You can use a example like when I used Tiburoncin.